pontoon/machinery/views.py

```diff
@@ -63,7 +64,7 @@ def translation_memory(request):
             suggestions[entry['target']]['count'] += 1
     except DataError as e:
         # Catches 'argument exceeds the maximum length of 255 bytes' Error
-        return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
+        return HttpResponse(status=501, reason='Not Implemented: {error}'.format(error=e))
```

Comment:
Whoops, my bad - text should not be in the `reason` attribute, it should be passed as the first positional argument, followed by `status`.

`return HttpResponse('Not Implemented: {error}'.format(error=e), status=501)`